BubbleScan-AI/Bubble_Scan_AI.py

```diff
@@ -0,0 +1,413 @@
+import json
+import re
+import shutil
+import fitz  # PyMuPDF
+import os
+import cv2
+import numpy as np
+
+
+class Scantron95945:
```

Comment:
I would put Scantron95945 class into a Scantron95945.py file, to make it very obvious where to find this class.